.github/workflows/cd.yml

```diff
@@ -110,18 +81,18 @@ jobs:
       - name: release alpha npm packages to npmjs.org
         if: ${{ github.event_name == 'schedule' || (github.event_name == 'workflow_dispatch' && github.ref == 'refs/heads/dev' && github.event.inputs.preid == 'alpha')}}
         run: |
-          npx lerna version --no-private prerelease --preid=alpha.$(git rev-parse --short HEAD) --no-push --allow-branch dev --yes
+          npx lerna version prerelease --preid=alpha.$(git rev-parse --short HEAD) --no-push --allow-branch dev --yes
```

Comment:
why remove --no-private?

Reply:
private in package.json set true, will skip publish this package, but version is still work.
version with --no-private will skip version this package.
now templates add in lerna, lerna will version templates, but no need to publish it.